src/Microsoft.AspNet.Mvc.ModelBinding/Binders/ComplexModelDtoModelBinder.cs

```diff
@@ -22,13 +22,14 @@ public sealed class ComplexModelDtoModelBinder : IModelBinder
             var dto = (ComplexModelDto)bindingContext.Model;
             foreach (var propertyMetadata in dto.PropertyMetadata)
             {
-                    var propertyModelName = ModelBindingHelper.CreatePropertyModelName(
-                            bindingContext.ModelName,
-                            propertyMetadata.BinderModelName ?? propertyMetadata.PropertyName);
+                var propertyModelName = ModelBindingHelper.CreatePropertyModelName(
+                        bindingContext.ModelName,
+                        propertyMetadata.BinderModelName ?? propertyMetadata.PropertyName);
 
-                var propertyBindingContext = new ModelBindingContext(bindingContext,
-                                                                     propertyModelName,
-                                                                     propertyMetadata);
+                var propertyBindingContext = ModelBindingContext.GetChildModelBindingContext(
+                    bindingContext,
+                    propertyModelName,
+                    propertyMetadata);
```

Comment:
Still not getting why the following code isn't part of `ModelBindingContext.GetChildModelBindingContext(...)`

```
var propertyModelName = ModelBindingHelper.CreatePropertyModelName(
                        bindingContext.ModelName,
                        propertyMetadata.BinderModelName ?? propertyMetadata.PropertyName);
```